src/core/link-to-dfn.js

```diff
@@ -206,6 +230,22 @@ function shouldWrapByCode(dfn, term) {
   return false;
 }
 
+/**
+ * Find additional references that need to be looked up externally.
+ * Examples: a[data-cite="spec"], dfn[data-cite="spec"], dfn.externalDFN
+ */
+function findExplicitExternalLinks() {
+  const selector =
+    "a[data-cite]:not([data-cite='']):not([data-cite*='#']), " +
+    "dfn[data-cite]:not([data-cite='']):not([data-cite*='#'])";
+  return [...document.querySelectorAll(selector)]
+    .filter(el => {
+      const closest = /** @type {HTMLElement} */ (el.closest("[data-cite]"));
```

Comment:
Maybe we should keep type definitions in their own line? Inline is reducing readability IMO.

Reply:
Inlining caused needless multi-lining:

```
const additionalExternalLinks = [
    ...document.querySelectorAll(
      "a[data-cite]:not([data-cite='']):not([data-cite*='#']), " +
        "dfn[data-cite]:not([data-cite='']):not([data-cite*='#'])"
    ),
  ]
```

4 lines vs 6 lines. Hmm.

Reply:
Just tried some modification, hoping you like it 👍

Reply:
I meant inside the filter. Sorry for misunderstanding

``` diff
- const closest = /** @type {HTMLElement} */ (el.closest("[data-cite]"));
+ /** @type {HTMLElement} */ 
+ const closest = el.closest("[data-cite]");
```

Reply:
OMG, maybe a need some coffee.

That's unfortunately impossible because `@ts-check` will warn that `Element` cannot be assigned to `HTMLElement`. It's like:

```
var closest: HTMLElement = element; // type error!
```